### PR TITLE
Fix log-generator and logging-demo charts

### PR DESCRIPTION
**Pull request by pepov**
Signed-off-by: Peter Wilcsinszky <peter.wilcsinszky@axoflow.com>

### CODE REVIEW

File: charts/log-generator/templates/deployment.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```
Comment (genofire) [inline charts/log-generator/templates/deployment.yaml:30]: use appVersion in Chart.yaml without v prefix

```suggestion
          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default (printf "v%s" .Chart.AppVersion) }}"
```

Comment (genofire) [reply charts/log-generator/templates/deployment.yaml:30]: PS: Maybe the image tag should also not have prefix

File: charts/log-generator/values.yaml [hunk 2]
```diff
@@ -6,7 +6,8 @@ replicaCount: 1
 
 image:
   repository: ghcr.io/kube-logging/log-generator
-  tag: v0.4.1
+  # set tag to override default appVersion
+  # tag:
```
Comment (genofire) [inline charts/log-generator/values.yaml:10]: keep, but empty for finding in README
```suggestion
  tag: ""
```

**Review by genofire — COMMENTED** [review]
please remove `charts/logging-demo/charts/*.tgz`

File: charts/log-generator/templates/deployment.yaml (hunk 1 above)
Comment (pepov) [reply charts/log-generator/templates/deployment.yaml:30]: I'm not sure I understand this.
```
appVersion: v0.5.0
```

appVersion has prefix

Comment (genofire) [reply charts/log-generator/templates/deployment.yaml:30]: hope it is clear now, with the comment in the new PR?

**Pull request merged**